### PR TITLE
Get rid of subgroups in inventory

### PR DESCRIPTION
**Pull request by Polina-Gubina**
Get rid of subgroups in inventory

### CODE REVIEW

File: inventory/extended/main.yaml [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    watcher_eu-de-01-vpc-b:
+      ansible_host: 10.10.0.2
+      ansible_ssh_user: linux
+      ansible_ssh_private_key_file: "{{ local_private_key }}"
+      az: eu-de-01
+      name: "watcher_eu-de-01-vpc-b"
+      role: watcher
+      vpc: vpc_b
+      scenarios: []
+      secgroup_tcp_ports: ["22", "80", "443", "2003", "2004", "2023", "2024", "8126"]
```
Comment (anton-sidelnikov) [inline inventory/extended/main.yaml]: ["22", "80", "443"]

File: inventory/extended/main.yaml [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+      secgroup_udp_ports: ["8126"]
```
Comment (anton-sidelnikov) [inline inventory/extended/main.yaml]: []

File: playbooks/lb_monitoring_destroy.yaml [hunk 3]
```diff
@@ -7,9 +7,15 @@
     - ./vars/lb_monitoring.yaml
   roles:
     - loadbalancer
-    - ecs_node
   tasks:
-    - name: Delete object from swift
+    - name: Include variables required for lm monitoring infra
```
Comment (anton-sidelnikov) [inline playbooks/lb_monitoring_destroy.yaml]: lb not lm

File: playbooks/lb_monitoring_setup.yaml [hunk 4]
```diff
@@ -92,9 +73,33 @@
...
             server_name: "http"
             extra_parameters: "{{ extra_parameters_http }}"
-          - listen: "443 ssl"
-            server_name: "https"
-            extra_parameters: "{{ extra_parameters_https }}"
+
+- name: Create key pair for lb monitoring and including required variables
+  hosts: local
+  tasks:
+    - name: Include variables required for lm monitoring infra
```
Comment (anton-sidelnikov) [inline playbooks/lb_monitoring_setup.yaml]: lm monitoring -> lb monitoring

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: playbooks/public_router.yaml [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+- name: Setting facts about created infra
+  set_fact:
+    main_network_id: "{{ net.network.id }}"
+    main_subnet_id: "{{ subnet.subnet.id }}"
+    main_router_id: "{{ router.router.id }}"
+
+- name: Setting facts about created infra
+  set_fact:
+    vpcs_info: "{{ vpcs_info | default({}) | \combine( {vpc_name: \
```
Comment (anton-sidelnikov) [inline playbooks/public_router.yaml]: strange thing

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

**Review by anton-sidelnikov — DISMISSED** [review]

**Comment by lego963** [comment]
recheck

**Comment by lego963** [comment]
recheck

**Review by anton-sidelnikov — APPROVED** [review]

**Pull request merged**